engines/query-sparql/test/QuerySparql-test.ts

```diff
@@ -573,6 +573,43 @@ describe('System test: QuerySparql', () => {
         expect((await arrayifyStream(await result.execute()))).toEqual([]);
       });
     });
+
+    // TODO: better name here before merge.
```

Comment:
TODO

Reply:
indeed. given my low SPARQL familiarity, i do not have a great idea. i'm enthusiastic about helping, and also i know my limits. could you suggest something please? it would actually help me understand the nature of this issue better too, and i'd love to keep contributing more in the future.

Reply:
Something like `'filter expression inside an optional'` should be fine.